tests/Fakturoid/ClientTest.php

```diff
@@ -452,7 +452,18 @@ public function testGetVatReports()
         $reports = $f->getVatReports(2021);
     }
 
-    /* Event */
+	/* Number lines */
+
+	public function testGetNumberLines()
+	{
+		$requester = $this->createMock('Fakturoid\Requester');
+		$requester->method('run')->willReturn(null);
+
+		$f = new Client('test', 'test@example.org', 'api-key', 'Test <test@example.org>', array('requester' => $requester));
+		$users = $f->getNumberLines();
+	}
+
+	/* Event */
```

Comment:
Poprosím opravit zanoření - používáme 4 mezery.

Reply:
Mělo by to být v pořádku.